src/RssBuilder.php

```diff
@@ -50,7 +50,7 @@ public function build(array $emails) : string
         foreach ($emails as $email) {
             $item = $this->dom->createElement('item');
             $this->addTextNode('title', $email->getSubject(), $item);
-            $this->addTextNode('link', $this->host . $email->getUrl(), $item);
+            $this->addTextNode('link', $this->host . '/message/' . $item['number'], $item);
```

Comment:
`$item` is a DomNode, I don't see why you do `['number']` on it 🤔 that shouldn't really work

Maybe it should be `$email->getNumber()` instead?

(ping @agallou who may have a better idea for this code)